circularprogressview/src/main/java/com/github/rahatarmanahmed/cpv/CircularProgressView.java

```diff
@@ -47,10 +50,11 @@ public CircularProgressView(Context context, AttributeSet attrs, int defStyle) {
 
     protected void init(AttributeSet attrs, int defStyle) {
         // Load attributes
-        final TypedArray a = getContext().obtainStyledAttributes(
+        typedArray = getContext().obtainStyledAttributes(
                 attrs, R.styleable.CircularProgressView, defStyle, 0);
 
-        a.recycle();
+
+        typedArray.recycle();
```

Comment:
Yo this code is broke as hell:

Not sure why you made typedArray a class member, it shouldn't really be accessed outside of init.

However I do notice my mistake in these lines that basically do nothing https://github.com/rahatarmanahmed/CircularProgressView/blob/master/circularprogressview/src/main/java/com/github/rahatarmanahmed/cpv/CircularProgressView.java#L50-L53

So what I want you to do is delete lines 53-57 in init(), and revert initAttributes back to the way it was, using a local TypedArray (but with your new options included).